docs/source/concepts/bento.rst

```diff
@@ -796,76 +796,70 @@ will build a new image on top of the base_image with the following steps:
         bentoml containerize iris_classifier:latest --platform=linux/amd64
 
 
-Docker Template (Danger Zone)
-"""""""""""""""""""""""""""""
+Dockerfile Template (Beta)
```

Comment:
Should we move this into an advanced guide?

Reply:
no.